src/ingestion-server/server/parameter.ts

```diff
@@ -241,7 +241,7 @@ export function createStackParameters(scope: Construct, props: {deliverToKinesis
           'logS3Bucket and logS3Prefix cannot be empty when enableApplicationLoadBalancerAccessLog=Yes',
       },
     ],
-  });
+  }).overrideLogicalId('logS3BucketAndEnableLogRule');
```

Comment:
It's unnecessary, we already suppressed this false positive finding.

Reply:
Rollbacked